modules/combined/tests/phase_field_fracture/crack2d_nonsplit.i

```diff
@@ -0,0 +1,163 @@
+[Mesh]
+  type = FileMesh
+  file = crack_mesh.e
+  second_order = true
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  order = SECOND
```

Comment:
looks like we have to use Hermite polynomial for nonsplit form because the second order laplacian term in the weak form.